test/unit/payment_icon_test.rb

```diff
@@ -23,6 +23,12 @@ class PaymentIconTest < ActiveSupport::TestCase
     end
   end
 
+  test 'Every payment icon record has a unique "name"' do
+    names = PaymentIcon.all.map(&:name)
+    duplicates = names.detect {|e| names.rindex(e) != names.index(e) }
```

Comment:
Good idea to add this!
Possible alternative assertion: `assert names.uniq.length < names.length`

Reply:
Though, maybe then you wouldn't know the duplicates..

Reply:
We wouldn't be able to identify the duplicates though (we'd only _know_ there are duplicate entries). 🤔